File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Internal/AllTests.cs
/* Copyright (C) 2004 - 2007  db4objects Inc.  http://www.db4o.com */

using System;
using Db4oUnit.Extensions;
using Db4objects.Db4o.Tests.Common.Internal;

namespace Db4objects.Db4o.Tests.Common.Internal
{
	public class AllTests : Db4oTestSuite
	{
		protected override Type[] TestCases()
		{
			return new Type[] { typeof(SerializerTestCase) };
		}
	}
}

File: db4oj.tests/src/com/db4o/db4ounit/common/internal/AllTests.java
/* Copyright (C) 2007 db4objects Inc. http://www.db4o.com */

package com.db4o.db4ounit.common.internal;

import db4ounit.extensions.*;


public class AllTests extends Db4oTestSuite {
	
	protected Class[] testCases() {
		return new Class[] {
				SerializerTestCase.class,
		};
	}

}
